router/core/flushwriter.go

```diff
@@ -122,6 +121,18 @@ func GetFlushWriter(ctx *resolve.Context, variables []byte, r *http.Request, w h
 		setSubscriptionHeaders(w)
 	}
 
+	w.WriteHeader(http.StatusOK)
```

Comment:
I'd a comment. This is important to respond early to the upgrade right?

Reply:
delete